### PR TITLE
TM1-7 Premium CardView UI

### PR DESCRIPTION
**Pull request by svlilyas**
:white_check_mark: Premium CardView UI add to ProfileFragment

### CODE REVIEW

File: app/src/main/res/layout/fragment_profile.xml [hunk 1]
```diff
@@ -94,11 +94,79 @@
...
+                android:orientation="vertical"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toStartOf="@+id/navigateIv"
+                app:layout_constraintStart_toEndOf="@+id/premiumIv"
+                app:layout_constraintTop_toTopOf="parent">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Join Premium!"
```
Comment (Feyzarkt) [inline app/src/main/res/layout/fragment_profile.xml:136]: Hardcoded string! should use @string resource

File: app/src/main/res/layout/fragment_profile.xml [hunk 2]
```diff
@@ -94,11 +94,79 @@
...
+                    android:textColor="@color/red"
+                    android:textFontWeight="700"
+                    android:textSize="20sp" />
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="5dp"
+                    android:text="Enjoy watching Full-HD movies, without restrictions and without ads"
```
Comment (Feyzarkt) [inline app/src/main/res/layout/fragment_profile.xml]: Hardcoded string! should use @string resource

File: app/src/main/res/layout/fragment_profile.xml (hunk 1 above)
Comment (Feyzarkt) [inline app/src/main/res/layout/fragment_profile.xml]: Hardcoded string! should use @string resource

**Review by Feyzarkt — COMMENTED** [review]
Good job 🙌

File: app/src/main/res/layout/fragment_profile.xml (hunk 2 above)
Comment (svlilyas) [reply app/src/main/res/layout/fragment_profile.xml]: Have missed them done ✅ , thansks for feedback.

File: app/src/main/res/layout/fragment_profile.xml (hunk 1 above)
Comment (svlilyas) [reply app/src/main/res/layout/fragment_profile.xml]: Done ✅

**Review by Feyzarkt — APPROVED** [review]
Excellent 👍

File: app/src/main/res/layout/fragment_profile.xml [hunk 3]
```diff
@@ -94,11 +94,79 @@
...
+                android:id="@+id/premiumIv"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:contentDescription="@string/app_name"
+                android:src="@drawable/ic_premium"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <LinearLayout
```
Comment (oliver-basoglu) [inline app/src/main/res/layout/fragment_profile.xml:123]: nit: Next time please do not nest viewGroups if you can avoid it. Above constraintLayout is enoguh to build this UI

**Review by oliver-basoglu — APPROVED** [review]
Great work :)

File: app/src/main/res/layout/fragment_profile.xml (hunk 3 above)
Comment (svlilyas) [reply app/src/main/res/layout/fragment_profile.xml:123]: I could not handle the situation, When you said I remembered about chainStyle (spread, packed), 
was it the right way to do so?

**Comment by svlilyas** [comment]
Bana atanan bir iş değildi. Çalışma amaçlı yaptım.

**Pull request closed without merging**